perses/rjmc/geometry.py

```diff
@@ -450,8 +451,6 @@ def _logp_propose(self, top_proposal, old_positions, beta, new_positions=None, d
         if self._storage:
             self._storage.write_object("{}_proposal_order".format(direction), proposal_order_tool, iteration=self.nproposed)
 
-        platform_name = 'CUDA'
```

Comment:
@ijpulidos : I don't think we want to make this fix here. I think we want to open a separate PR to fix this, as both the protein mutation and small molecule tests need this fix. Then we'll want to open/merge a PR that speeds up the protein mutation tests, then finally we can merge this PR #1065.

Reply:
Fair enough, I agree that helps keeping track of things. It was just easier for me to test the consequences of this fix in this same branch, but I agree we should probably want a different one for this fix for bookkeeping.